scripts/Fig6-FigS7_bulk-analysis.R

```diff
@@ -94,6 +176,93 @@ bulk_table <- bulk_table |>
     n_ora_genesets        = NA_integer_ # the sum of this quantity isn't relevant
   )
 
+# Scatterplot panels -----------------------------------------------------------
+
+# Read in data
+model_data_df <- data_files |>
+  purrr::map(readr::read_tsv) |>
+  purrr::list_rbind(names_to = "scpca_project_id") |>
+  dplyr::select(scpca_project_id, bulk, pseudobulk)
+
+# Prepare data frame for plotting
+model_data_df <- model_data_df |>
+  # join with the number of samples in each project
+  dplyr::inner_join(
+    dplyr::select(bulk_table, scpca_project_id, n_samples_used)
+  ) |>
+  # create faceting columns with the sample count info
+  dplyr::mutate(
+    project_facet = glue::glue("{scpca_project_id} (N={n_samples_used})")
+  )
+
+
+# Main text panel:
+scatter_panel_main <- model_data_df |>
+  dplyr::filter(scpca_project_id %in% main_projects) |>
+  plot_scatterplot()
+  
+
+# SI panel:
+scatter_panel_si <- model_data_df |>
+  dplyr::filter(scpca_project_id %in% si_projects) |>
+  plot_scatterplot()
+
+
+
+
+# Odds ratio panels ------------------------------------------------------------
+
+# Read in odds ratio results and filter to significant only
+odds_df <- odds_files |>
+  purrr::map(readr::read_tsv) |>
+  purrr::list_rbind(names_to = "scpca_project_id") |>
+  # filter to significant at 0.05
+  dplyr::filter(p_adj_bh <= alpha)
+
+# Prepare data frame for plotting  
+odds_df <- odds_df |>
+  # join with the number of samples in each project
+  dplyr::inner_join(
+    dplyr::select(bulk_table, scpca_project_id, n_samples_used)
+  ) |>
+  # create columns for plotting 
+  dplyr::mutate(
+    # faceting with the sample count info
+    project_facet = glue::glue("{scpca_project_id} (N={n_samples_used})"), 
+    # wrapped gene set names for space
+    geneset_cell_type = stringr::str_wrap(geneset_cell_type, 20)
+  )
+
+# Main text panel:
+odds_panel_main <- odds_df |>
+  dplyr::filter(scpca_project_id %in% main_projects) |>
+  plot_odds_ratios()
+
+
+# SI panel:
+odds_panel_si <- odds_df |>
+  dplyr::filter(scpca_project_id %in% si_projects) |>
+  plot_odds_ratios()
```

Comment:
We have a little bit of an issue with the spacing of the `mononuclear phagocyte` and `dendritic cell labels` for `SCPCP000017`. I wonder if a slightly different angle just for the supplemental figure is the solution?

Reply:
I made the angle 45 all around, and I think it looks better in both supp and main panels.